_data/simple-icons.json

```diff
@@ -75,6 +75,11 @@
             "hex": "FF5A5F",
             "source": "https://www.airbnb.com"
         },
+        {
+            "title": "Go Language",
+            "hex": "6AD7E5",
```

Comment:
The [official favicon](https://golang.org/doc/gopher/favicon.svg) uses `#76E1FE` - although it changes wildly from image to image

Reply:
I'm fine either way, I feel like the one you linked is more correct based on my affiliation with the brand. I'll change it :+1: